actors/install-actor.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, type Page } from '@playwright/test';
+import { MainPage } from '../pages/main-page';
+import { ConfirmInstallationPage } from '../pages/confirm-installation-page';
+import { InstallationProgressPage } from '../pages/installing-page';
+import { CongratulationsPage } from '../pages/installation-finsihed-page';
+
+export class InstallActor {
+    readonly page: Page;
+    readonly mainPage: MainPage;
+
+    constructor(page: Page, mainPage: MainPage) {
+        this.page = page;
+        this.mainPage = mainPage;
```

Comment:
looks like you are missing to pass the confirmation page and the installation progress to the actor